next-hardhat/utils/noir/noirBrowser.ts

```diff
@@ -29,7 +29,7 @@ export class NoirBrowser {
     this.acirBuffer = Buffer.from(circuit.bytecode, 'base64');
```

Comment:
@kevaundray any changes on how we handle circuit bytecode required per the recent changes re en/decoding base64 on both Noir's and barretenberg's sides?

Reply:
Link to [headless-test/src/index.ts](https://github.com/AztecProtocol/barretenberg/blob/master/acir_tests/headless-test/src/index.ts) for quick ref.

Reply:
I made all of the relevant changes in this barretenberg PR: https://github.com/AztecProtocol/aztec-packages/commit/7fffd1680d6246f64ee4d4ca965b9764c6c0ebb3

If you follow the examples, it will show how to handle the base64 encoding; essentially, base64 is only in the json file, bb.js and bb require you to base64 decode it before passing it to them